net/xtables-addons/Makefile

```diff
@@ -8,7 +8,7 @@ include $(INCLUDE_DIR)/kernel.mk
 
 PKG_NAME:=xtables-addons
 PKG_VERSION:=3.27
-PKG_RELEASE:=1
+PKG_RELEASE:=2
```

Comment:
After committing https://github.com/openwrt/packages/pull/26368 change from 2 to 3

Reply:
Sure, I'll wait until the other #26368 has been merged.